### ISSUE TITLE
API插件的问题

### ISSUE THREAD

**Issue by duguhanghai** [Blankj/AndroidUtilCode#1505]
## 描述需求

u should impl these apis  
使用api的时候报和这个错误,但是我们其他组件是动态添加的,有可能存在不一定非要实现接口的问题,

**Comment by Blankj** [comment]
其他组件不存在那就加个 mock 接口就行

**Comment by duguhanghai** [comment]
不是,嗯就是假设我在common模块定义个一个class A继承ApiUtils.BaseApi(),然后在组件1实现 class B继承A,但是组件1是运维进行配置选择是否会依赖,那么就会 存在上述问题,在这个情况下我应该怎么处理,多谢了

**Comment by duguhanghai** [comment]
> 其他组件不存在那就加个 mock 接口就行

不是,嗯就是假设我在common模块定义个一个class A继承ApiUtils.BaseApi(),然后在组件1实现 class B继承A,但是组件1是运维进行配置选择是否会依赖,那么就会 存在上述问题,在这个情况下我应该怎么处理,多谢了

**Comment by Blankj** [comment]
在 app 顶层写个 继承 A 的类，注解中加 mock = true，那样就不会为空了。。

**Comment by cervy** [comment]
或根据情况配置编译时是否abort。。
